tests/run_tests.py

```diff
@@ -188,23 +205,38 @@ def main():
 
     # system
     if any(i in ['all', 'system'] for i in args.tests):
-        test_results['system'] = run_system(args.system, vic_exe, data_dir,
-                                            os.path.join(out_dir, 'system'),
-                                            args.driver)
+        if len(args.driver) == 1:  # if only one driver
+            driver = args.driver[0]
+            test_results['system'] = run_system(args.system, vic_exe, data_dir,
+                                                os.path.join(out_dir,
+                                                             'system'),
+                                                driver)
+        else:  # if multiple drivers
+            list_driver = args.driver
+            test_results['system'] = run_system(args.system, list_vic_exe,
+                                                data_dir,
+                                                os.path.join(out_dir,
+                                                             'system'),
+                                                list_driver)
```

Comment:
Maybe just pass all the drivers in and then choose how to work with them in `run_system()`.